js/content/community.js

```diff
@@ -714,7 +714,20 @@ let ProfileHomePageClass = (function(){
 
     ProfileHomePageClass.prototype.addTwitchInfo = async function() {
 
-        let search = document.querySelector(".profile_summary a[href*='twitch.tv/'], .customtext_showcase a[href*='twitch.tv/']");
+        if (!SyncedStorage.get('profile_showcase_twitch')) { return; }
+
+        if (User.isSignedIn && !SyncedStorage.get('profile_showcase_own_twitch')) {
+            if (window.location.pathname == User.profilePath) {
```

Comment:
We should have something like isOwnProfile, if we don't yet, because we tend to check that quite often.